### ISSUE TITLE
Add TOTP authentication to management shell

### ISSUE THREAD

**Issue by bluecmd** [u-root/u-bmc#147]
In parallel to https://github.com/u-root/u-bmc/issues/136 it would be nice to have some sort of authentication on the console by default.

Using TOTP should be a pretty good way of not having to add passwords but still have something that is useful and re-usable across a fleet of machines.

**Comment by bluecmd** [comment]
TOTP appears to be symmetrical which is not great. Might need to revisit if we want this. We can do something like an asymmetrical URL-based login with offline validation, but that's pretty complex if we don't have the need.
